rules/DowngradePhp80/Rector/MethodCall/DowngradeReflectionGetAttributesRector.php

```diff
@@ -83,6 +83,7 @@ public function refactor(Node $node): Ternary|null|int
         if (
             ! $this->isObjectType($node->var, new ObjectType('Reflector'))
             && ! $this->isObjectType($node->var, new ObjectType('ReflectionClass'))
+            && ! $this->isObjectType($node->var, new ObjectType('ReflectionProperty'))
```

Comment:
@TomasVotruba the real solution is probably patch the `NodeTypeResolver::isMatchingUnionType()` instead, so it can work on its object or the child for nullable.